ruby/sorting_array_performance_in_ruby.md

```diff
@@ -0,0 +1,32 @@
+You can sort an array in a various way. Just keep in mind that there is a difference in performance.
```

Comment:
Please update README.md as well
(e.g: https://github.com/Tapjoy/til/pull/17/files#diff-04c6e90faac2675aa89e2176d2eec7d8R45)

Reply:
Updated README.md
Thanks for the review 👍

Reply:
👍